cmake/std/atdm/contributed/weaver/all_supported_builds.sh

```diff
@@ -1,4 +1,4 @@
-export ATDM_CONFIG_CTEST_S_BUILD_NAME_PREFIX=Trilinos-atdm-waterman-
+export ATDM_CONFIG_CTEST_S_BUILD_NAME_PREFIX=Trilinos-atdm-weaver-
```

Comment:
I think we can delete this file now that 'weaver' is a custom/contributed configuration.  It really does not have any meaning if there are no automated builds running.

Reply:
Removed this file